README.md

```diff
@@ -54,5 +55,9 @@ Just run `git-copy-history` again and it will add only the new commits.
 git-copy-history from ../local-repo
 git push
 ```
+## Update your profile settings
+
+If you have private repository and did not check this box: go [here](https://github.com/settings/profile) and check the box:
+![Your Profile](https://user-images.githubusercontent.com/41075647/62783212-9c4a9280-bac4-11e9-9cf8-906c0a6918fe.png)
```

Comment:
Good! Thank you! Could you provide an image link with higher resolution?

Reply:
Is it good enough?
https://user-images.githubusercontent.com/41075647/62835932-98ef0c80-bc66-11e9-82a3-ce83d93801bf.png

Reply:
That's great! But could you use `img` tag with twice less width than the original image? Since as for now it's just a huge image at the bottom.

Reply:
Please, choose on your own taste
![option1](https://user-images.githubusercontent.com/41075647/62838582-16c31000-bc87-11e9-80cc-0258a27e9138.png)
![option2](https://user-images.githubusercontent.com/41075647/62838583-17f43d00-bc87-11e9-913e-1318de29ef92.png)

Reply:
Updated with gif image.